Description: Enable vertical edge tapping by default even if we enable two-finger scrolling as well.
Bug-Ubuntu: https://launchpad.net/bugs/320632

Index: xserver-xorg-input-synaptics-1.8.2/src/synaptics.c
===================================================================
--- xserver-xorg-input-synaptics-1.8.2.orig/src/synaptics.c
+++ xserver-xorg-input-synaptics-1.8.2/src/synaptics.c
@@ -633,8 +633,8 @@ set_default_parameters(InputInfoPtr pInf
     clickFinger2 = (priv->has_right || priv->has_middle) ? 1 : 3;
     clickFinger3 = (priv->has_right || priv->has_middle) ? 1 : 2;
 
-    /* Enable vert edge scroll if we can't detect doubletap */
-    vertEdgeScroll = priv->has_double ? FALSE : TRUE;
+    /* Enable vert edge scroll */
+    vertEdgeScroll = TRUE;
     horizEdgeScroll = FALSE;
 
     /* Enable twofinger scroll if we can detect doubletap */
